### PR TITLE
ENH: Add bundle profile plotting function

### PR DESCRIPTION
**Pull request by richford**
I have been using this bundle profile plotting function in my own work and thought it might be useful to others. It will plot mean bundle profiles with optional grouping by a target variable.

Example output:
[hbn_age_profiles_after_site_combat_md.pdf](https://github.com/richford/AFQ-Insight/files/6165334/hbn_age_profiles_after_site_combat_md.pdf)

### CODE REVIEW

File: afqinsight/plot.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+import seaborn as sns
+
+from collections import OrderedDict
+from groupyr.transform import GroupExtractor
+from tqdm.auto import tqdm
+
+from .utils import BUNDLE_MAT_2_PYTHON
+
+
+POSITIONS = OrderedDict(
```
Comment (arokem) [inline afqinsight/plot.py:15]: You intentionally kept the same positions as the one used in pyAFQ? That's different from what you have in the plot that you had.

Comment (arokem) [reply afqinsight/plot.py:15]: Also, note that this dict has positions for both the waypoint ROI-based and the Recobundles-based bundles.

File: afqinsight/plot.py [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+            df = df.melt(id_vars=id_vars, var_name="nodeID", value_name=metric)
+            df["nodeID"] = df["nodeID"].astype(int)
+            tract_stats[tid] = df
+
+        # Arrange the bundles into a grid
+        bgcolor = "white"
+        positions = POSITIONS.copy()
+        positions["FA"] = (0, 0)
+        positions["FP"] = (0, 3)
```
Comment (arokem) [inline afqinsight/plot.py:182]: Oh - I see! Wouldn't you rather do that at the top, now that you are not importing this from pyAFQ?

**Review by arokem — COMMENTED** [review]
Looks good! I noticed a couple of things, but nothing crucial.

File: afqinsight/plot.py (hunk 1 above)
Comment (richford) [reply afqinsight/plot.py:15]: Thanks! Resolved in the most recent commit.

File: afqinsight/plot.py (hunk 2 above)
Comment (richford) [reply afqinsight/plot.py]: Good call, resolved in the most recent commit.

**Comment by richford** [comment]
Ugh, all checks passed except for coveralls. But I really don't want to write image comparison unit tests à la matplotlib. What do you think @arokem, should we just merge without writing unit tests. Or is there a more light-weight testing paradigm for plotting functions?

**Comment by arokem** [comment]
:shipit:

**Pull request merged**